match/docs/openapi/orchestrator/query.yaml

```
@@ -4,6 +4,12 @@ post:
     - "Match"
   summary: "Search for all matching PII records"
   description: "Queries all state databases for any PII records that are an exact match to the last name, date of birth, and social security number of persons provided in the request."
+  parameters:
+    - in: header
+      name: From
+      schema:
+        type: string
+      description: "The username of the individual making the request"
```

```suggestion
      description: "As in the HTTP/1.1 RFC, used for logging purposes as a means for identifying the source of invalid or unwanted requests. The interpretation of this field is that the request is being performed on behalf of the state government-affiliated person whose email address (or username) is specified here. It is not used for authentication or authorization."
```
We should clarify some more; I've copied and modified the `From` description from the HTTP RFC in the suggestion above.